### ISSUE TITLE
Constants uppercase report regression

### ISSUE THREAD

**Issue by dereuromark** [squizlabs/PHP_CodeSniffer#2050]
```php
protected function getTableReflection(): ClassReflection
```

Invalid report:

> Constants must be uppercase; expected CLASSREFLECTION

See https://travis-ci.org/cakephp/cakephp/jobs/389179510#L646

**Comment by dereuromark** [comment]
Seems like a small regression that we can however fix on our end just removing the old sniff that used to work up until now: https://github.com/cakephp/cakephp-codesniffer/pull/207

So if you think there is nothing else that should/can be done for BC, then you can also close.

**Comment by gsherwood** [comment]
This is probably because that string used to be a T_RETURN_TYPE token but is now T_STRING, so the sniff would need a small change to ensure the string does not come after a colon. You might have had the same problem before if you ever used a namespace for a return type because tokenizing them was such a mess, so there were T_STRING tokens in there as well.

I'll close this as the break is expected and you've found another sniff to replace your one as well.

For more info, have a read of issue https://github.com/squizlabs/PHP_CodeSniffer/issues/1527
